pennylane/ops/op_math/condition.py

```diff
@@ -100,7 +101,111 @@ def adjoint(self):
         return Conditional(self.meas_val, self.base.adjoint())
 
 
-def cond(condition, true_fn, false_fn=None, elifs=()):
+class CondCallable:  # pylint:disable=too-few-public-methods
+    """Base class to represent a conditional function with boolean predicates.
+
+    Args:
+        condition (bool): a conditional expression
+        true_fn (callable): The function to apply if ``condition`` is ``True``
+        false_fn (callable): The function to apply if ``condition`` is ``False``
+        elifs (List(Tuple(bool, callable))): A list of (bool, elif_fn) clauses.
+
+    Passing ``true_fn``, ``false_fn``, and ``elifs`` on initialization

```

Comment:
It looks like `true_fn` is not optional in the init method

Reply:
```suggestion
    Passing ``false_fn`` and ``elifs`` on initialization
```